server/src/test/resources/config/elasticsearch.yml

```diff
@@ -1,3 +1,2 @@
 
 yaml.config.exists: true
-
```

Comment:
can you revert this change

Reply:
Ah missed this, removed.

Reply:
It seems that it's still there ? ;)

Reply:
Hmm that's strange, might be my IDE setup problem. Anyway I checked out and committed the previous version of this file before my changes, and I don't see this change in `Files Changed` tab anymore. so it should be good now.